src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -52,6 +52,7 @@ type Props = {
     changeSharedLinkPermissionLevel: (
         newPermissionLevel: permissionLevelType,
     ) => Promise<{ permissionLevel: permissionLevelType }>,
+    children: React.Node,
```

Comment:
I would not call it children, since thats a generic for all/any children.
Probably should be `classification?: ClassificationInfo` and should use https://github.com/box/box-ui-elements/tree/master/src/features/classification

Reply:
updated branch! yet I didn't put the classification as optional, because it's passed in as object from upstream